homeassistant/components/airzone/climate.py

```diff
@@ -124,8 +124,6 @@ def __init__(
         self._attr_unique_id = f"{self._attr_unique_id}_{system_zone_id}"
         self._attr_supported_features = ClimateEntityFeature.TARGET_TEMPERATURE
         self._attr_target_temperature_step = API_TEMPERATURE_STEP
-        self._attr_max_temp = self.get_airzone_value(AZD_TEMP_MAX)
```

Comment:
Think these needs to be set during init as well

Reply:
> Think these needs to be set during init as well

_async_update_attrs is called at the end of the init function, so it's called:
https://github.com/home-assistant/core/blob/11857241544f8b445fd897a2ff9611fc9705fbb7/homeassistant/components/airzone/climate.py#L113-L139

Reply:
Perhaps you can also add a test to see that it works as expected

Reply:
> Perhaps you can also add a test to see that it works as expected

Sure, I can do that this afternoon :)

Reply:
> Perhaps you can also add a test to see that it works as expected

Done in 9df40522fbc1e96691278d365cd15ae89dace2c6